src/client/utils/dom.js

```diff
@@ -499,6 +499,13 @@ export function isWindow (instance) {
         return true;
 
     try {
+        // NOTE: A window object received from MessageEvent.target has strange behavior into js debugger:
```

Comment:
Change comment to:

This is special hack for `MSEdge` and `IE` browsers.
`InstanceToString` call result has a strange values for a `MessageEvent.target` property:
`[object DispHTMLWindow2]` for IE11
`[object Object]` for `MSEdge`.

Reply:
Add separate tests for `isWindow` function